aws/resource_aws_dx_private_virtual_interface.go

```diff
@@ -219,3 +235,22 @@ func dxPrivateVirtualInterfaceWaitUntilAvailable(d *schema.ResourceData, conn *d
 			directconnect.VirtualInterfaceStateDown,
 		})
 }
+
+func IntInSlice(valid []int) schema.SchemaValidateFunc {
```

Comment:
Again, haven't fully tested this yet but would this be a good candidate to move to the core validation library?

Reply:
Yes, I think it would but given the churn in the codebase with v0.12 it is probably best to wait until that settles down.

Reply:
Probably better to move this to [`validators.go`](https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/validators.go) and rename it to something like `validateIntInSlice`.